app/controllers/api/v1/deliveries_controller.rb

```diff
@@ -63,17 +63,67 @@ def destroy
       render json: {}
     end
 
-    private
-
-    def serializer
-      Api::V1::DeliverySerializer
+    api :POST, '/confirm_delivery', "Confirm Delivery with address, contact and schedule details"
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.